### PR TITLE
Provide 6.69.1 version

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@xhochy and @marcelotrevisani 
Could you please review?

File: recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# !/usr/bin/env bash
+
+set +ex
+
+cmake ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release -S "$SRC_DIR" -B "$SRC_DIR/build" || exit 1
+cmake --build "$SRC_DIR/build" --config Release --parallel $CPU_COUNT --verbose || exit 1
```
Comment (xhochy) [inline recipe/build.sh:6]: Why are you adding these `|| exit 1` everywhere?
```suggestion
cmake ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release -S "$SRC_DIR" -B "$SRC_DIR/build"
cmake --build "$SRC_DIR/build" --config Release --parallel $CPU_COUNT --verbose
```

File: recipe/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# !/usr/bin/env bash
+
+set +ex
```
Comment (xhochy) [inline recipe/build.sh:3]:
```suggestion
set -ex
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by ilya-lavrenov** [comment]
@xhochy could you please have a look once again?

**Review by xhochy — APPROVED** [review]

**Pull request merged**